### DISCUSSION TITLE
Probe fails to extend every other time while using the Auto Build Mesh feature.

### DISCUSSION THREAD

**Discussion by stikosek** [mriscoc/Ender3V2S1#1350] [Q&A] [answered]
Hello, today i installed a 3D touch bed leveling probe on my Ender 3 V2.
The probe works really well, extends and retracts OK and homes correctly. However, when using the Auto Build Mesh feature,
the probe extends correctly on the first point, but when the printer moves to probe to the next one and takes a measurement, it fails to extend and crashes into a bed - only stopping because of the original Z limit switch, the cycle then repeats. Once it extends and takes a measurement, then it crashes without extending.
I am honestly quite stumped by this. When sending a command manually to the probe, it never once failed to extend. Only when using the Auto Build Mesh feature. I have searched this github page for some time, but failed to see someone with the same problem
I am using the 4.2.2 BLTUBL-MPC 202401025 firmware. My printer is, except for the probe, stock, if that helps. Any help would be appreciated

**Comment by mriscoc** [comment, marked as answer]
Some probes are incompatible with the HS mode, try disabling it in probe settings menu.

![probe-settings-2](https://github.com/user-attachments/assets/830f6639-0e72-4bff-bc55-f23fc7e6dc84)

**Reply by stikosek** [reply]
Hi, thanks a lot for the response, that was it!
I read that part of the wiki page and tried this before, but i think i was "testing" it with the M48 Probe test, which didn't work due to a second misconfiguration that i then fixed but didn't retry with the mesh leveling, so the problem was my own stupidity.  
I really appreciate what you are doing here, your firmware is hands down the best i ever tried, so thanks a lot for all the hard work.
